napari/utils/colormaps/colormap.py

```diff
@@ -236,13 +243,27 @@ def _selection_as_minimum_dtype(self, dtype: np.dtype) -> int:
 
 
 class LabelColormap(LabelColormapBase):
-    """Colormap that shuffles values before mapping to colors.
+    """Colormap color values using cycle of colors .
 
     Attributes
     ----------
-    seed : float
+    colors : ColorArray
+        Colors to be used for mapping.
+        For values above the number of colors,
+        the colors will be cycled.
     use_selection : bool
+        Whether map only selected label.
+        If `True` only selected label will be mapped to not transparent color.
     selection : int
+        The selected label.
+    background_value : int
+        Which value should be treated as a background
+        and mapped to transparent color.
+    interpolation : Literal['zero']
+        required by implementation, please do not set value
```

Comment:
Can we somehow freeze this value, rather than requiring users to cooperate?

Reply:
```diff
diff --git a/napari/utils/colormaps/colormap.py b/napari/utils/colormaps/colormap.py
index d48bcffa3..d43a984de 100644
--- a/napari/utils/colormaps/colormap.py
+++ b/napari/utils/colormaps/colormap.py
@@ -176,7 +176,7 @@ class LabelColormapBase(Colormap):
     background_value: int = 0
     interpolation: Literal[
         ColormapInterpolationMode.ZERO
-    ] = ColormapInterpolationMode.ZERO
+    ] = Field(ColormapInterpolationMode.ZERO, frozen=True)
     _cache_mapping: Dict[Tuple[np.dtype, np.dtype], np.ndarray] = PrivateAttr(
         default={}
     )
```
Does this do what you'd like? With this change, setting the field to anything else would raise a `ValidationError`:
```
ValidationError: 1 validation error for LabelColormap
interpolation
  unexpected value; permitted: <ColormapInterpolationMode.ZERO: 'zero'> (type=value_error.const; given=linear; permitted=(<ColormapInterpolationMode.ZERO: 'zero'>,))
  ```